pyxem/signals/diffraction2d.py

```diff
@@ -1008,6 +1012,14 @@ def center_direct_beam(
         electron diffraction pattern and translate it to the center of the
         image square.
 
+        The direct beam position can either be passed to the method with the shifts
```

Comment:
Could we move this into the description of the `shifts` argument?

Reply:
Done! Together with a slight rewrite, of the docstring. Including two examples on how to use the function.